### PR TITLE
The Documentation Examples Tests are now working on VS 2017 

### PR DESCRIPTION
**Pull request by loneosama**
You have to run the command for some tests to pass. The approvals tests specifically.

### CODE REVIEW

**Comment by loneosama** [comment]
ok I enterted a wrong commit message I meant to say that shouldmatch approved now works with core

**Comment by loneosama** [comment]
@JosephWoodward  is there something wrong with the pull request?

**Comment by hemantkd** [comment]
@loneosama Does all 1304 tests in the Solution pass for from the ShouldMatchApproved Scenarios? I'm asking because in the AppVeyor, not all tests are run.

Visual Studio's Test Explorer only show 675 tests but Resharper's Unit Test Sessions window show 1306 tests. I can confirm all the former pass just as the AppVeyor suggests.

File: src/DocumentationExamples/CodeExamples/ShouldMatchApprovedExamples.ApprovedFileDoesNotExist.exceptionText.approved.txt [hunk 1]
```diff
@@ -1,6 +1,6 @@
 To approve the changes run this command:
-copy /Y "C:\PathToCode\shouldly\src\DocumentationExamples\ShouldMatchApprovedExamples.ApprovedFileDoesNotExist.received.txt" "C:\PathToCode\shouldly\src\DocumentationExamples\ShouldMatchApprovedExamples.ApprovedFileDoesNotExist.approved.txt"
+copy /Y "C:\Users\OsamaLone\source\repos\shouldlyosama\src\DocumentationExamples\ShouldMatchApprovedExamples.ApprovedFileDoesNotExist.received.txt" "C:\Users\OsamaLone\source\repos\shouldlyosama\src\DocumentationExamples\ShouldMatchApprovedExamples.ApprovedFileDoesNotExist.approved.txt"
```
Comment (dennisroche) [inline src/DocumentationExamples/CodeExamples/ShouldMatchApprovedExamples.ApprovedFileDoesNotExist.exceptionText.approved.txt]: These files should be reverted as they contain paths from your environment

Comment (loneosama) [reply src/DocumentationExamples/CodeExamples/ShouldMatchApprovedExamples.ApprovedFileDoesNotExist.exceptionText.approved.txt]: i tried doing it but for some reason they were not reverted. The thing is i dont have time as i am currently busy with my exams. But when i do get it i ll update them or you can replace these by pulling this pull request or by going to my [repository](https://github.com/loneosama/shouldly) to pull it. Or when i get time i ll make changes in your repository if needed and you can send the pull request.
Thanks

**Comment by loneosama** [comment]
thanks @hayer for the reverted paths

**Comment by hayer** [comment]
@dennisroche anything more you want done before this can be merged? 😄

**Comment by dennisroche** [comment]
@hayer i'll pull the branch locally this morning and test. i'm surprised that all was required to get approval tests working was adding the `ShouldMatchApproved` define. 

i did the port to `netstandard2.0` and we left it disabled for a reason.

**Comment by dennisroche** [comment]
The test project doesn't have `ShouldMatchApproved` defined so those tests aren't being executed 🤔. Given the PR title, what were the changes required to make the `DocumentationExamples` tests run?

**Comment by loneosama** [comment]
I basically added a DebugType tag in csproj to Full. That was the reason that it was not running the tests

**Comment by dennisroche** [comment]
I would split the PR; make this just fix the documentation tests and move the `ShouldMatchApprove` to another PR.

**Comment by loneosama** [comment]
@dennisroche  I have split the PRs into to other pull requests

**Pull request closed without merging**